com.github.louis77.tuner.json

```diff
@@ -0,0 +1,67 @@
+{
+    "app-id": "com.github.louis77.tuner",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno-19.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "com.github.louis77.tuner",
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*",
+        "--share=network",
+        "--metadata=X-DConf=migrate-path=/com/github/louis77/tuner/",
+        "--socket=pulseaudio",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.gnome.SettingsDaemon.MediaKeys",
+        "--talk-name=org.mpris.MediaPlayer2",
+        "--own-name=org.mpris.MediaPlayer2.Tuner"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "/share/gtk-doc",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "taglib",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DBUILD_SHARED_LIBS=ON",
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "sources": [{
+                "type": "archive",
+                "url": "https://github.com/taglib/taglib/archive/v1.11.1.tar.gz",
+                "sha256": "b6d1a5a610aae6ff39d93de5efd0fdc787aa9e9dc1e7026fa4c961b26563526b"
+            }]
+        },
+        {
+            "name": "tuner",
+            "buildsystem": "meson",
+            "build-options": {
```

Comment:
there's a config-opts key as well, you don't need to set build-options to set them.

Reply:
Thank you for the hint!